vql/server/elastic.go

```diff
@@ -204,9 +205,14 @@ func append_row_to_buffer(
 		row_dict.Delete("_index")
 	}
 
-	meta := []byte(fmt.Sprintf(
-		`{ "index" : {"_id" : "%d", "_type": "%s", "_index": "%s" } }%s`,
-		id, arg.Type, index, "\n"))
+    var meta []byte = []byte(fmt.Sprintf(`{ "index" : {"_id" : "%d", "_type": "%s", "_index": "%s"} }%s`,
```

Comment:
Maybe avoid allocating the string twice?
```
var meta []byte 
if arg.PipeLine == "" {
   ....
} else {
   ...
}
```

Reply:
Done